server/utils/licenceStatus.js

```diff
@@ -350,12 +350,13 @@ function getDmApproval(licence) {
     };
 
     const decision = getIn(licence, ['approval', 'release', 'decision']);
-    const reason = getIn(licence, ['approval', 'release', 'reason']);
+    const reasons = flatten([getIn(licence, ['approval', 'release', 'reason'])]);
 
     return {
         approved: decision === 'Yes',
         refused: decision === 'No',
-        refusalReason: refusalReasons[reason],
+        refusalReason: reasons.reduce((reasons, reason, index) =>
+            `${reasons}${index > 0 ? ', ' : ''}${index > 0 ? refusalReasons[reason].toLowerCase() : refusalReasons[reason]}`, ''),
         approval: isEmpty(decision) ? taskStates.UNSTARTED : taskStates.DONE
```

Comment:
what if you made refusalReasons all lowercase, do a join, and capitalise initial letter?

Reply:
Good call